rust/src/lib.rs

```diff
@@ -0,0 +1,61 @@
+#[macro_use(lazy_static)]
+extern crate lazy_static;
+
+pub mod do_username {
+	use rand::Rng;
+	fn to_title_case(s: &str) -> String {
+		let mut c = s.chars();
+		match c.next() {
+			None => String::new(),
+			Some(f) => f.to_uppercase().chain(c).collect(),
+		}
+	}
+	
+	fn random(arr: Vec<&str>) -> &str {
+		let mut rng = rand::thread_rng();
+		let r = rng.gen_range(0, arr.len());
+		arr[r]
+	}
+	fn up_format(arr: Vec<&str>) -> String {
+		let mut new_arr = vec![];
+		for elem in arr {
+			new_arr.push(to_title_case(elem))
+		}
+		new_arr.join("")
+	}
+	lazy_static! {
+		pub static ref SEA_CREATURES: std::vec::Vec<&'static str> = vec!["walrus", "seal", "fish", "shark", "clam", "coral", "whale", "crab", "lobster", "starfish", "eel", "dolphin", "squid", "jellyfish", "ray", "shrimp", "mantaRay", "angler", "snorkler", "scubaDiver", "urchin", "anemone", "morel", "axolotl"];
+		pub static ref SEA_OBJECTS: std::vec::Vec<&'static str> = vec!["boatShip", "submarine", "yacht", "dinghy", "raft", "kelp", "seaweed", "anchor"];
+		pub static ref ADJECTIVE_DESCRIPTORS: std::vec::Vec<&'static str> = vec!["cute", "adorable", "lovable", "happy", "sandy", "bubbly", "friendly", "floating", "drifting"];
+		pub static ref SIZE_DESCRIPTORS: std::vec::Vec<&'static str> = vec!["large", "big", "small", "giant", "massive", "tiny", "little", "yuge"];
+		pub static ref CREATURE_DESCRIPTORS: std::vec::Vec<&'static str> = vec!["swimming", "sleeping", "eating", "hiding"];
+		pub static ref SEA_LIST: Vec<&'static str> = SEA_OBJECTS.iter().cloned().chain(SEA_CREATURES.iter().cloned()).collect();
+		pub static ref DESCRIPTORS: Vec<&'static str> = ADJECTIVE_DESCRIPTORS.iter().cloned().chain(SIZE_DESCRIPTORS.iter().cloned()).collect();
+		pub static ref COLORS: std::vec::Vec<&'static str> = vec!["blue", "blueGreen", "darkCyan", "electricBlue", "greenBlue", "lightCyan", "lightSeaGreen", "seaGreen", "turquoise", "aqua", "aquamarine", "teal", "cyan", "gray", "darkBlue", "cerulean", "azure", "lapis", "navy"];
+	}
+	pub fn generate(size: Option<usize>) -> String {
```

Comment:
```suggestion

	pub fn generate(size: Option<usize>) -> String {
```
Consistent whitespace